crates/ruff_python_codegen/src/generator.rs

```diff
@@ -67,6 +68,8 @@ pub struct Generator<'a> {
     indent: &'a Indentation,
     /// The line ending to use.
     line_ending: LineEnding,
+    /// Forced quote style to use.
```

Comment:
The comment here is inaccurate. Setting `quote` doesn't force the generator to use the given quote everywhere. For example, setting `Quote::Single` for `f'test{f"nested"}'` doesn't change the quotes of the inner f-string. I also recommend adding a test showcasing this behavior

That's why I'd call this field `preferred_quote`. We should also extend the documentation to explain in more detail what it does (When `None`, the generator preserves the quote style when possible. When `Some`, the generator prefers the set quote style, disregarding the quote style used in the source)

Reply:
that's a better name for sure